css/index.css

```
@@ -0,0 +1,355 @@
+        :root {
+            /* body color */
+            --body: #CFE5FF;
+
+            /* primary color */
+            --primary_100: #3692FF;
+            --primary_200: #1967D6;
+            --primary_300: #1251AA;
+
+            /* error color */
+            --error: #F74747;
+
+            /* Secondary color */
+            --gray_50: #F9FAFB;
+            --gray_100: #F3F4F6;
+            --gray_200: #E5E7EB;
+            --gray_400: #9CA3AF;
+            --gray_500: #6B7880;
+            --gray_600: #4B5563;
+            --gray_700: #374151;
+            --gray_800: #1F2937;
+            --gray_900: #111827;
+        }
+
+        body {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+            font-family: 'pretendard', 'Abhaya Libre ExtraBold' -apple-system, BlinkMacSystemFont, system-ui, Roboto,
```

💊 제안
대체폰트가 기획이나 디자인상 명시된 것이 아니니 없어도 될 것 같아요!

```suggestion
            font-family: 'pretendard'
```